db/migrate/20171226134252_add_lead_source_to_oauth_applications.rb

```diff
@@ -0,0 +1,6 @@
+class AddLeadSourceToOauthApplications < ActiveRecord::Migration
+  def change
+    add_column :oauth_applications, :lead_application_source, :string,
+                                    null: false, default: ''
```

Comment:
Not a huge fan of the blank string as a default. I'd maybe leave the column as nullable, if some apps have no lead_application_source. But I guess if Salesforce expects a blank string that may be ok.